pages/index.js

```
@@ -1,7 +1,112 @@
+import * as CONSTANTS from "../src/components/constants";
+
+// Inserted global variable definitions
+let DEBUG = process.env.NODE_ENV === "development";
+DEBUG = false;
+
+let READING_SPEED = CONSTANTS.DEFAULT_READING_SPEED; // in words-per-minute (wpm)
+let START_COLOR = CONSTANTS.START_COLOR;
+let STOP_COLOR = CONSTANTS.STOP_COLOR;
+
+const initialContent = DEBUG ? CONSTANTS.EPICTETUS : CONSTANTS.INTRO_TEXT;
+let scrollingEnabled = DEBUG ? false : true;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)